src/mappings/handlers/multisigCallHandler.ts

```diff
@@ -84,34 +83,18 @@ async function getTransaction(visitedCall: VisitedCall): Promise<MultisigOperati
 
   const operationId = generateOperationId(callHash, multisigAccountId, blockCreated, indexCreated);
 
-  let operationData = {}
-
-  if (call) {
-    try {
-      operationData = call.toHuman() as {} | string;
-
-      if (typeof operationData === 'string') {
-        const tx = api.tx(call.toHex())
-        operationData = tx.method.toHuman() as {};
-      }
-    } catch (error) {
-      logger.info(error)
-    }
-  }
-
   const newOperation = await MultisigOperation.create({
     ...existingOperation,
     id: operationId,
+    section: call?.section,
+    method: call?.method,
+    callData: call?.toHex(),
     callHash,
     status: existingOperation?.status || OperationStatus.pending,
     accountId: multisigAccountId,
     depositor: u8aToHex(decodeAddress(visitedCall.origin)),
     blockCreated,
     indexCreated,
-    ...(call ? {
-      callData: call.toHex(),
-      ...operationData
-    } : {}),
     timestamp: timestamp(visitedCall.extrinsic.block)
```

Comment:
typescript argues about
```
Type 'number' is not assignable to type 'bigint'.ts(2322)
```

Reply:
@stepanLav Try to run `yarn codegen` again